packages/app/obojobo-document-engine/src/scripts/viewer/components/inline-nav-button/inline-nav-button.js

```diff
@@ -0,0 +1,55 @@
+import './inline-nav-button.scss'
+
+import React from 'react'
+import { connect } from 'react-redux'
+
+
+const inlineNavButton = (props) => {
+
+    let navIndex
+    let content
+    let disabled
+	switch (props.type) {
+		case 'prev':
+            navIndex = props.currentNavIndex - 1;
```

Comment:
I know this is just a work in progress but I'd recommend you get a prettier extension for your IDE of choice and have it run on saving the file, that should auto-indent files